gh_pages/_source/session2/Services.md

```diff
@@ -87,13 +87,13 @@ Your goal is to create a more intricate system of nodes:
       )
       ```
 
-   3. Edit the `package.xml` file to add the appropriate build/run dependencies:
+   3. Edit the `package.xml` file to add the appropriate build/exec dependencies:
      
       ``` xml
       <build_depend>message_generation</build_depend>
-      <run_depend>message_runtime</run_depend>
+      <exec_depend>message_runtime</exec_depend>
       <build_depend>geometry_msgs</build_depend>
-      <run_depend>geometry_msgs</run_depend>
+      <exec_depend>geometry_msgs</exec_depend>
```

Comment:
Thanks for bringing this to our attention, however this is only valid for packages with the **format=2** tag.  At some point we'll update our package.xml files to this format.